core/shared/src/main/scala/zio/logging/internal/json/SmallBooleanStack.scala

```diff
@@ -0,0 +1,35 @@
+package zio.logging.internal.json
+
+final class SmallBooleanStack(
+  private val data: Long
+) extends AnyVal {
+  /* 64 - stackSize reserved for index store -> 6 bits are enough to represent 58 states */
+  @inline final private def stackSize      = 58
+  @inline final private def idxIncrement   = 0x400000000000000L // 1L << stackSize
+  @inline final private def undefinedValue = 0xffffffffffffffffL
```

Comment:
maybe those 3 defs should be in companion object as val

Reply:
Not relevant anymore.